app/models/participation.rb

```diff
@@ -9,7 +9,8 @@ class Participation < ApplicationRecord
   validates :participant_id, uniqueness: { scope: :edition_id }
 
   scope :not_deleted, -> { where(deleted: false).includes(:edition).order('editions.date desc') }
-  scope :with_points, -> { where('total_points > 0').count }
+  scope :with_points, -> { where('total_points > 0') }
+  scope :no_participants_with_points, -> { with_points.count }
```

Comment:
in this case I would use **number_of** instead of **no**

I was reading **no** as negative and it was not making sense

what do you think?

Reply:
Hm, it might be an option, yes.